src/neo/Ledger/Blockchain.cs

```diff
@@ -302,7 +302,6 @@ private void OnInventory(IInventory inventory, bool relay = true)
             };
             if (relay && rr.Result == VerifyResult.Succeed)
                 system.LocalNode.Tell(new LocalNode.RelayDirectly { Inventory = inventory });
-            Sender.Tell(rr);
```

Comment:
This can't be removed.

Reply:
This message doesn't seen to be used by any code..

Reply:
It is widely used in clients.

Reply:
Understood, I have restored corresponding logic, plz have a check